Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Use an official Python runtime as a parent image
-FROM python:3.7
```

Comment:
Could we take this opportunity and upgrade to 3.8.x?

Reply:
I am happy to